eng/scripts/Language-Settings.ps1

```diff
@@ -19,7 +19,17 @@ function Get-java-PackageInfoFromRepo ($pkgPath, $serviceDirectory, $pkgName)
 
     if ($projectPkgName -eq $pkgName)
     {
-        return [PackageProps]::new($pkgName, $pkgVersion.ToString(), $pkgPath, $serviceDirectory, $pkgGroup)
+        $pkgProp = [PackageProps]::new($pkgName, $pkgVersion.ToString(), $pkgPath, $serviceDirectory, $pkgGroup)
+        if ($projectPkgName -match "mgmt" -or $projectPkgName -match "resourcemanager")
+        {
+          $pkgProp.SdkType = "mgmt"
+        }
+        else
```

Comment:
We should also add a SdkType for spring package for any package name that matches "spring".

Reply:
For reference here are the spring list https://github.com/Azure/azure-sdk/blob/master/_data/releases/latest/java-packages.csv#L89-L119.